src/pages/app/posting_project_user/ProfilePage.js

```diff
@@ -43,6 +43,20 @@ const imageURL =
 const loremText =
   "Khi Beyond Birthday lấy mạng nạn nhân thứ ba của mình, hắn định làm một thử nghiệm—để xem có cách nào làm một con người chết vì xuất huyết trong mà không cần phải hủy hoại một bộ phận cơ thể nào hay không. Cụ thể, hắn cho nạn nhân chìm vào trạng thái vô thức bằng thuốc mê; ...";
 
+//The button to add a project
+function AddProjectButton(props) {
```

Comment:
```suggestion
function _addProjectButton(props) {
```

Reply:
What, why? :))) It's a component so it should be named like that.

Reply:
My bad.

Reply:
Yea, so I can resolve this conversation now right?